src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -21,21 +21,31 @@
 
 (def valid-commands-str (str/join ", " (sort (vals command-names))))
 
+(defn coerce-to-int
+  [s]
+  (if (number? s) s (Integer. s)))
```

Comment:
just to tie this off, neither of those works here. I also let payload-to-body-string be with respect to your suggested refactor because rob has some code on top of this that removes the function anyway.